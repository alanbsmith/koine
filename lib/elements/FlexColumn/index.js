import styled from 'styled-components';
import { rem } from 'polished';

const FlexColumn = styled.div`
  box-sizing: border-box;
  display: flex;
  flex: none;
  padding: ${props => rem(props.theme.dimensions.basePad)};
`;

export default FlexColumn;
