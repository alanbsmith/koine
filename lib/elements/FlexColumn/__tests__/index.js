import React from 'react';
import FlexColumn from '../index';

import { renderWithTheme } from '__tests__/helpers';

describe('FlexColumn Element', () => {
  it('renders correctly', () => {
    const tree = renderWithTheme(
      <FlexColumn />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
