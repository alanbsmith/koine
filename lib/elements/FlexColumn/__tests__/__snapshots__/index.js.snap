// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FlexColumn Element renders correctly 1`] = `
<div
  className="sc-bdVaJa faXlxK"
/>
`;
